[FISH-1390] Payara 5.28.1 Release Notes (#151)
payara/Payara-Enterprise-Documentation: docs/modules/ROOT/pages/release-notes/release-notes-28-1.adoc

= Payara Enterprise 5.28.1 Release Notes

== Supported APIs and Applications

 * Java EE 8 Applications
 * Jakarta EE 8
 * MicroProfile 3.3

== Notes

=== Incompatibility with the latest JVM / JDK versions and Monitoring

> IMPORTANT: Payara Platform products do run correctly on JDK 8u292+ and 11.0.11+ due to the removal of SSL/TLS  1.0 and SSL/TLS 1.1 protocols in these versions. The JMX service doesn't start properly which means there are no monitoring data available.

Reviewer: This paragraph is very confusing to me.

If I understand it correctly that Payara Platform runs OK on the latest JDK versions and only JMS service doesn't start, then it should clearly state that:

* Due to removal of TLS 1.0 and 1.1, monitoring data will not be available because the JMX service will fail to start and will not produce monitoring data.
* All the other functionality of Payara Platform except monitoring works correctly on the latest JDKs and Payara Platform can run on them in production without any issues

Author: Since we can't test on the latest JDK (the TCK and other tests fail) we cannot say that "other functionality" works correctly here

> IMPORTANT: Payara Platform products do run correctly on JDK 8u292, 11.0.11 or later. Due to removal of SSL/TLS 1.0 and 1.1 protocols, monitoring data will not be available because the JMX service fails to start, and will not produce monitoring data. Other issues due to the missing protocols may also be present.

Reviewer: ```suggestion
IMPORTANT: Due to removal of SSL/TLS 1.0 and 1.1 protocols in the latest OpenJDK versions, monitoring data will not be available on JDK 8u292 and 11.0.11 or later because the JMX service fails to start and will not produce monitoring data. Other issues due to the missing protocols may also be present.
```
